appveyor.yml

```diff
@@ -42,7 +42,7 @@ build_script:
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
   - Rscript -e "install.packages('covr', repos = 'http://cran.us.r-project.org')"
-  - Rscript -e "install.packages('ospsuite.utils', repos = 'http://cran.us.r-project.org', type = 'source')"
+  - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/sdy8a68mbyxpctx2/artifacts/ospsuite.utils_1.2.0.9000.zip', repos = NULL, type = 'binary')"

```

Comment:
Always use the latest version?

Reply:
This needs to change tto use the latest version

Reply:
The latest commit in _ospsuite.utils_ didn't work:
https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-rutils/builds/43044129/artifacts

There is nothing at 
https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip